src/xo-app/vm/tab-disks.js

```diff
@@ -453,6 +452,83 @@ export default class TabDisks extends Component {
       newDisk
     } = this.state
 
+    const COLUMNS = [
```

Comment:
Try to move `COLUMNS` out of the class body, you can use `SortedTable`'s `userData` property if you want access to something.